core/src/main/java/org/testcontainers/containers/ContainerDef.java

```diff
@@ -1,304 +1,9 @@
 package org.testcontainers.containers;
 
-import com.github.dockerjava.api.command.CreateContainerCmd;
-import com.github.dockerjava.api.model.Bind;
-import com.github.dockerjava.api.model.ExposedPort;
-import com.github.dockerjava.api.model.HostConfig;
-import com.github.dockerjava.api.model.InternetProtocol;
-import com.github.dockerjava.api.model.PortBinding;
-import com.github.dockerjava.api.model.Ports;
-import lombok.Getter;
-import lombok.extern.slf4j.Slf4j;
-import org.testcontainers.DockerClientFactory;
-import org.testcontainers.UnstableAPI;
-import org.testcontainers.containers.wait.strategy.WaitStrategy;
-import org.testcontainers.images.RemoteDockerImage;
-import org.testcontainers.utility.DockerImageName;
-import org.testcontainers.utility.TestcontainersConfiguration;
+class ContainerDef extends BaseContainerDef {
```

Comment:
Why do we need `ContainerDef` and `BaseContainerDef`? Wouldn't either be enough?